Libplanet/Net/Protocols/KademliaProtocol.cs

```diff
@@ -196,19 +194,21 @@ public ImmutableList<BoundPeer> PeersToBroadcast
                     tasks.Add(FindPeerAsync(
                         new Address(buffer),
                         null,
-                        0,
+                        -1,
```

Comment:
What does means `-1`?

Reply:
It indicates that the operation is not being limited by `depth`. Please take a look at <a href="https://github.com/planetarium/libplanet/pull/525/files#diff-ae527d5afcdf41da4a57b594cab3cfefR528">line 528</a>.

Reply:
It would be good if such behavior is documented.